compiler-rt/test/asan/TestCases/fakeframe-right-redzone.cpp

```diff
@@ -1,5 +1,6 @@
 // RUN: %clangxx_asan -O0 %s -o %t && %env_asan_opts=detect_stack_use_after_return=1 not %run %t 2>&1 | FileCheck %s
 // RUN: %clangxx_asan -O0 %s -o %t -fsanitize-address-use-after-return=always && not %run %t 2>&1 | FileCheck %s
+// UNSUPPORTED: MSVC
```

Comment:
nit: here msvc is capitalized (i.e. "MSVC") but in your other file it's not (i.e. "msvc").
It seems the codebase mostly uses the latter form. I recommend using the latter form in both files